README.md

```diff
@@ -33,8 +33,15 @@ assert.deepEqual(nestedList, decoded)
 
 ## CLI
 
-`rlp decode <hex string>`  
-`rlp encode <json String>`
+`rlp encode <json string>`
+`rlp decode <hex string>`
+
+Examples:
+
+- `rlp encode 5` => `0x05`
```

Comment:
I think we need to be careful with this example since it's encoding an integer and not a string and the CLI is specifically intended to encode/decode strings.  When I first read this example, I expected `rlp decode 0x05` => 5 where it actually produces `"\u0005"`.  If you try to encode/decode integers much larger than this, you get all sorts of unexpected results if you're anticipating this to encode and decode integers.  

I think we should remove this line and then add a clear note that this is only intended to encode/decode strings and not integers.

Reply:
very good point, will address, thanks!